data/mods/Aftershock/vehicles/afs_vehicle_overrides.json

```diff
@@ -0,0 +1,343 @@
+      { "y": 1, "x": 2, "parts": [ "hdframe_vertical_2", "seat", "hdroof" ] },
+      { "y": 0, "x": 2, "parts": [ "hdframe_vertical_2", "aisle_horizontal", "hdroof" ] },
+      { "y": -1, "x": 2, "parts": [ "hdframe_vertical_2", "seat", "hdroof" ] },
+      { "y": -2, "x": 2, "parts": [ "hdframe_vertical", "hdboard_vertical", "plating_military" ] },
+      { "y": -2, "x": 2, "parts": [ "wheel_armor" ] },
+      { "y": 2, "x": 1, "parts": [ "hdframe_vertical", "hdboard_vertical", "plating_military" ] },
+      { "y": 2, "x": 1, "parts": [ "wheel_armor" ] },
+      { "y": 1, "x": 1, "parts": [ "hdframe_vertical_2", "seat", "hdroof" ] },
+      { "y": 0, "x": 1, "parts": [ "hdframe_vertical_2", "aisle_horizontal", "hdroof" ] },
+      { "y": -1, "x": 1, "parts": [ "hdframe_vertical_2", "seat", "hdroof" ] },
+      { "y": -2, "x": 1, "parts": [ "hdframe_vertical", "hdboard_vertical", "plating_military" ] },
+      { "y": -2, "x": 1, "parts": [ "wheel_armor" ] },
+      { "y": 2, "x": 0, "parts": [ "hdframe_vertical", "hdboard_vertical", "plating_military" ] },
+      { "y": 2, "x": 0, "parts": [ "wheel_armor" ] },
+      { "y": 1, "x": 0, "parts": [ "hdframe_vertical_2", "seat", "hdroof" ] },
+      { "y": 0, "x": 0, "parts": [ "hdframe_vertical_2", "aisle_horizontal", "hdroof" ] },
+      { "y": -1, "x": 0, "parts": [ "hdframe_vertical_2", "seat", "hdroof" ] },
+      { "y": -2, "x": 0, "parts": [ "hdframe_vertical", "hdboard_vertical", "plating_military" ] },
+      { "y": -2, "x": 0, "parts": [ "wheel_armor" ] },
+      { "y": 2, "x": -1, "parts": [ "hdframe_vertical", "hdboard_vertical", "plating_military" ] },
+      { "y": 1, "x": -1, "parts": [ "hdframe_vertical_2", "seat", "hdroof" ] },
+      { "y": 0, "x": -1, "parts": [ "hdframe_vertical_2", "aisle_horizontal", "hdroof" ] },
+      { "y": -1, "x": -1, "parts": [ "hdframe_vertical_2", "seat", "hdroof" ] },
+      { "y": -2, "x": -1, "parts": [ "hdframe_vertical", "hdboard_vertical", "plating_military" ] },
+      { "y": 2, "x": -2, "parts": [ "hdframe_se", "hdboard_se", "muffler", "plating_military" ] },
+      { "y": 1, "x": -2, "parts": [ "hdframe_horizontal", "hdboard_horizontal", "plating_military" ] },
+      { "y": 0, "x": -2, "parts": [ "hdframe_horizontal", "hddoor_opaque", "omnicam", "plating_military" ] },
+      { "y": -1, "x": -2, "parts": [ "hdframe_horizontal", "hdboard_horizontal", "plating_military" ] },
+      { "y": -2, "x": -2, "parts": [ "hdframe_sw", "hdboard_sw", "muffler", "plating_military" ] },
+      { "y": 2, "x": -3, "parts": [ { "fuel": "diesel", "part": "external_tank" } ] },
+      { "y": -2, "x": -3, "parts": [ { "fuel": "diesel", "part": "external_tank" } ] }
+    ]
+  },
+  {
+    "id": "truck_trailer",
+    "type": "vehicle",
+    "name": "Truck Trailer",
+    "blueprint": [ [ "--OO------" ], [ "||OO||||||" ], [ "+|++|=|++|" ], [ "+|++|=|++|" ], [ "||OO||||||" ], [ "--OO------" ] ],
+    "parts": [
+      { "x": 0, "y": 0, "part": "hdframe_vertical_2" },
```

Comment:
Please consolidate multiple parts on the same tile with `"parts": [ ]` like you did with the APCs above.

Reply:
This was actually just a copypaste from the normal vehicles, with instances of `cargo_space` replaced with `afs_hauling_space`. I'll look into changing it to be less icky.